### ISSUE TITLE
Search form

### ISSUE THREAD

**Issue by alexroseb** [abrenon/opigno-a11y#1]
## Overview
The search bar is theoretically part of the tab index, but `:focus` doesn't show up when it's tabbed to and it doesn't expand when accessed via tab. If you catch it at the right time you can type in a query but that's not visible until you mouse over and the bar expands. It might work for screenreaders but is completely inaccessible for keyboard-only users. 
## Thoughts
We may just need to have the search bar always open, which isn't terrible (it's just hidden for _aesthetics_). 
## Goals

- [ ] Add `:focus` attribute to the search button itself
- [ ] Show search box when focus shifted to it (backup: always have search open)
